src/app/component/CurrencySwitcher/CurrencySwitcher.component.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+import React, { PureComponent } from 'react';
+import { setCurrency, getCurrency } from 'Util/Currency/Current';
+import PropTypes from 'prop-types';
+import Field from 'Component/Field/Field.container';
+
+class CurrencySwitcherComponent extends PureComponent {
```

Comment:
`Component` post-fix is not required. Please remove it. `CurrencySwitcher` is enough.